test/steward_admin_change.js

```diff
@@ -10,46 +10,67 @@ const {
   multiPatronageCalculator,
   waitTillBeginningOfSecond,
   STEWARD_CONTRACT_NAME,
-  ERC721_CONTRACT_NAME
+  ERC20_CONTRACT_NAME,
+  ERC721_CONTRACT_NAME,
+  MINT_MANAGER_CONTRACT_NAME
 } = require("./helpers");
 
-const Artwork = artifacts.require(ERC721_CONTRACT_NAME);
+const ERC721token = artifacts.require(ERC721_CONTRACT_NAME);
 const WildcardSteward = artifacts.require(STEWARD_CONTRACT_NAME);
+const ERC20token = artifacts.require(ERC20_CONTRACT_NAME);
+const MintManager = artifacts.require(MINT_MANAGER_CONTRACT_NAME);
 
 const PATRONAGE_DENOMINATOR = "1";
 const patronageCalculator = multiPatronageCalculator(PATRONAGE_DENOMINATOR);
 
 contract("WildcardSteward owed", accounts => {
-  let artwork;
+  let erc721;
   let steward;
+  let erc20;
   const testTokenId1 = 1;
   const patronageNumerator = 12;
   const patronageDenominator = 1;
+  const tokenGenerationRate = 10; // should depend on token
   let testTokenURI = "test token uri";
 
   beforeEach(async () => {
-    artwork = await Artwork.new({ from: accounts[0] });
+    erc721 = await ERC721token.new({ from: accounts[0] });
```

Comment:
We should make this into a common function (the before each). It will make the code more maintainable, since almost same code in 6 places)

Reply:
TODO for next sprint? Test refactor doing this and other house cleaning on tests.

Reply:
Yeah, lets put it in the backlog. Doesn't have to be next sprint, but it mustn't be forgotten. :)